lib/mvi/service.rb

```diff
@@ -58,27 +60,37 @@ def find_profile(user)
 
     private
 
-    # TODO: Possibly consider adding Grafana Instrumentation here too
     def mvi_error_handler(user, e)
       case e
       when MVI::Errors::DuplicateRecords
-        log_message_to_sentry('MVI Duplicate Record', :warn)
+        log_info_and_errors('MVI Duplicate Record', :warn)
       when MVI::Errors::RecordNotFound
-        # Not going to log RecordNotFound
-        # NOTE: ICN based lookups do not return RecordNotFound. They return InvalidRequestError
-        # log_message_to_sentry('MVI Record Not Found', :warn)
-        nil
+        # Not going to log RecordNotFound to sentry, cloudwatch only.
+        log_info_and_errors('MVI Record Not Found')
```

Comment:
we're going to have a little trouble tying the logging context to the previous measure_info call (we can't do a single query that matches back toward another entry in the cloudwatch console). 

Best bet is to duplicate that info in the log statement.